client/src/game/ui/HanabiCard.ts

```diff
@@ -34,6 +34,8 @@ export default class HanabiCard extends Konva.Group implements NodeWithTooltip {
   bareName: string = '';
 
   tweening: boolean = false;
+  wasRecentlyTapped: boolean = false;
+  touchstartTimeout?: NodeJS.Timeout | null;
```

Comment:
1) type is wrong on this https://stackoverflow.com/questions/51040703/what-return-type-should-be-used-for-settimeout-in-typescript
2) pretty sure this needs to get reset on refresh